apps/optimizely/src/optimizely-client.js

```diff
@@ -25,39 +25,64 @@ export default class OptimizelyClient {
     this.onReauth();
   };
 
-  getProjects() {
-    return this.makeRequest('/projects');
-  }
-
-  getExperiment = (experimentId) => {
-    return this.makeRequest(`/experiments/${experimentId}`);
-  };
-
-  _getExperimentsPerPage = (perPage, page) => {
-    return this.makeRequest(
-      `/experiments?project_id=${
-        this.project
-      }&per_page=${perPage.toString()}&page=${page.toString()}`
-    );
-  };
-
-  getExperiments = async () => {
-    let experiments = [];
+  _getItemsPerPage = async (item) => {
+    let items = [];
     const PER_PAGE = 100;
-    const MAX_REQUESTS = 5;
+    const MAX_REQUESTS = 10;
 
     for (let i = 1; i <= MAX_REQUESTS; i++) {
-      const results = await this._getExperimentsPerPage(PER_PAGE, i);
-      experiments = [...experiments, ...results];
+      const results = await this.makeRequest(this._getItemsUrl(PER_PAGE, i, item));
+      items = [...items, ...results];
       if (results.length < PER_PAGE) {
         break;
       }
     }
 
-    experiments = experiments.filter((experiment) => {
-      return experiment.status !== 'archived';
+    if (item === 'experiment') {
+      items = items.filter((experiment) => {
+        return experiment.status !== 'archived';
+      });
+    }
+
+    items = items.sort((a, b) => {
```

Comment:
Awesome! I wonder if it might be better to sort by `created_at` or something of the like if it exists, to get the most recently created "experiments" at the top for UX 🤔 just a thought, otherwise this is great.

Reply:
Good point, there are options for `created` and `last_modified`. At this point, I'm going to say some sorting is better than no sorting and will go ahead with this fix. Maybe we will hear some feedback on sorting preferences from customers and then can address that